lib/instrumentation/langchain/tools.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const { LangChainTool } = require('../../llm-events/langchain')
+const common = require('./common')
+const {
+  AI: { LANGCHAIN }
+} = require('../../metrics/names')
+const runId = Symbol('runId')
```

Comment:
Will we need this in other LangChain instrumentations?

Reply:
I'm not sure yet. but I will move this symbol to `lib/symbols` regardless